client/src/app/site/users/components/user-detail/user-detail.component.ts

```diff
@@ -141,6 +142,11 @@ export class UserDetailComponent extends BaseModelContextComponent implements On
     }
 
     private loadUserById(): void {
+        // Prevent race conditions since this method is invoked two times.
```

Comment:
This is not the solution. Use observables. Some lines above are two calls to this function. You can merge both observables. Also awaiting this.request models can prevent the race condition (do it aditionally)